app/models/users/deleted_user.rb

```diff
@@ -24,6 +24,13 @@ def self.profile_image_url() = nil
     def self.path() = nil
     def self.tag_line() = nil
 
+    # @return [String]
+    #
+    # @see ApplicationRecord#class_name
+    def self.class_name
```

Comment:
Because this doesn't inherit from ApplicationRecord.  Also perhaps technically not needed, but added for completeness.